### PR TITLE
Rapatriate Inference types to moon

### PR DESCRIPTION
**Pull request by julien-c**

### CODE REVIEW

File: packages/tasks/src/model-data.ts [hunk 1]
```diff
@@ -27,8 +12,9 @@ export interface ModelData {
 	id: string;
 	/**
 	 * Whether or not to enable inference widget for this model
+	 * TODO(type it)
 	 */
-	inference: InferenceDisplayability;
+	inference: string;
```
Comment (coyotte508) [inline packages/tasks/src/model-data.ts:17]:
```suggestion
	inference: string | "Yes" | ...;
```

To keep some documentation while being lax?

Comment (julien-c) [reply packages/tasks/src/model-data.ts:17]: but see https://github.com/huggingface-internal/moon-landing/pull/10618 (internal) it's gonna change quite a bit

**Review by coyotte508 — APPROVED** [review]

**Comment by julien-c** [comment]
merging, even though i'm surprised that it doesn't break the (archived) widget build, given it uses this type

(we can just move the type into the archived widgets if it's simpler)

**Comment by coyotte508** [comment]
> merging, even though i'm surprised that it doesn't break the (archived) widget build, given it uses this type
> 
> (we can just move the type into the archived widgets if it's simpler)

widgets was removed from `pnpm-workspace.yml`, so it's not being built or anything in the CI

**Pull request merged**